parm/config/gfs/config.snowanl

```diff
@@ -9,6 +9,7 @@ echo "BEGIN: config.snowanl"
 source "${EXPDIR}/config.resources" snowanl
 
 export OBS_LIST="${PARMgfs}/gdas/snow/obs/lists/gdas_snow.yaml.j2"
+export GTS_LIST="${PARMgfs}/gdas/snow/obs/config/bufr2ioda_mapping.yaml.j2"
```

Comment:
I think I would be more descriptive here and name this `GTS_SNOW_STAGE_YAML` unless other reviewers find this too verbose

Reply:
Made changes as suggested.